### PR TITLE
fix(es/typescript): Strip class modifiers

### PR DESCRIPTION
**Pull request by magic-akari**
**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: crates/swc_fast_ts_strip/src/lib.rs [hunk 1]
```diff
@@ -441,6 +442,59 @@ impl TsStrip {
...
+                }
+                Token::Word(Word::Ident(IdentLike::Known(
+                    KnownIdent::Readonly
+                    | KnownIdent::Public
+                    | KnownIdent::Protected
+                    | KnownIdent::Private,
+                ))) => {
+                    self.add_replacement(*span);
+                }
+                Token::Word(Word::Ident(IdentLike::Other(o))) if *o == "override" => {
```
Comment (magic-akari) [inline crates/swc_fast_ts_strip/src/lib.rs:489]: Adding `override` to the `KnownIdent` is likely a wise move.

Comment (kdy1) [reply crates/swc_fast_ts_strip/src/lib.rs:489]: I think we should add it

Comment (magic-akari) [reply crates/swc_fast_ts_strip/src/lib.rs:489]: Let's add it to #9313.

Comment (kdy1) [reply crates/swc_fast_ts_strip/src/lib.rs:489]: Sounds good 👍

**Review by kdy1 — APPROVED** [review]
I'll merge once the publish pipeline ends

**Comment by kdy1** [comment]
I found that we can include this patch in 1.7.8

**Comment by magic-akari** [comment]
The ts-type strip still needs one or two additional fix.

**Comment by magic-akari** [comment]
> I found that we can include this patch in 1.7.8. Can you fix the CI failures?

patch-section.sh removed in c977c2bec2b0f9771031ed6ebc15bffb0f33eae0.
How to fix it?

**Comment by kdy1** [comment]
You can do `cargo install cargo-link2` and do `cargo link ../` from `./bindings`. Sorry, I forgot documenting it.

**Comment by magic-akari** [comment]
> error: the `#[global_allocator]` in swc_malloc conflicts with global allocator in: swc_malloc

The CI process has failed due to a conflict with swc_malloc. Could you help me? @kdy1

**Comment by kdy1** [comment]
I'll take on this

**Review by kdy1 — APPROVED** [review]
I restored `patch-sections.sh` for now. Sorry for my mistake and I'll fix the bug of `cargo-link` after publishing this change.

**Pull request merged**